test/test_RequestHandler.py

```diff
@@ -1,64 +1,100 @@
+import http
+import json
 import os
 import random
 import string
 import unittest
+import uuid
 
 import boto3
-import uuid
 from boto3.dynamodb.conditions import Key
 from moto import mock_dynamodb2
-import arrow as arrow
+
+from common.constants import Constants
+from common.encoders import encode_resource, encode_file_metadata, encode_files, encode_creator, encode_metadata
+from common.helpers import remove_none_values
+from data.creator import Creator
+from data.file import File
+from data.file_metadata import FileMetadata
+from data.metadata import Metadata
+from data.resource import Resource
+from data.title import Title
+
+
+def unittest_lambda_handler(event, context):
+    unittest.TextTestRunner().run(
+        unittest.TestLoader().loadTestsFromTestCase(TestHandlerCase))
+
+
+def remove_mock_database(dynamodb):
+    dynamodb.Table(os.environ[Constants.ENV_VAR_TABLE_NAME]).delete()
+
+
+def generate_mock_event(operation, resource):
+    body = Body(operation, resource)
+    body_value = json.dumps(body, default=encode_body)
+    return {
+        'body': body_value
+    }
 
 
 @mock_dynamodb2
 class TestHandlerCase(unittest.TestCase):
+    EXISTING_RESOURCE_IDENTIFIER = 'ebf20333-35a5-4a06-9c58-68ea688a9a8b'
+    EXISTING_RESOURCE_IDENTIFIER_MISSING_CREATED_DATE = 'acf20333-35a5-4a06-9c58-68ea688a9a9c'
```

Comment:
this is just an identifier, but it's name is indicating the later use. Is'nt it enough to call id1 and id2? maybe?

Reply:
not important